aws.js

```diff
@@ -1,24 +1,24 @@
-new (function() {
-	
+new(function () {
+
 	var ext = this;
-	
+
 	$.getScript('https://ceyhunozgun.github.io/awsAIScratchExtension/aws-sdk-2.270.1.js', initExtension);
 
 	var accessKeyId = '';
 	var secretAccessKey = '';
 
 	var polly;
 	var rekognition;
```

Comment:
Here you are declaring a variable which means it won't' have rekognition.compareFaces function with it. 
check with docs how to integrate rekognition @aswinkumar2019